components/nav-modal/nav.css

```diff
@@ -0,0 +1,81 @@
+:root {
+    --grey: grey;
+    --white-light: rgb(248, 244, 244);
+    --white: white;
+    --peacock: #084f70;
+    --blue: #1b9bff;
+
+}
+
+.nav-container-C {
+    border: 1px solid avr(--grey);
+    padding: 1.56rem 0.32rem 1.56rem 0.32rem;
+    width: 80%;
+    background-color: var(--white-light);
+}
+
+.navbar-comp {
+    color: var(--white);
+    width: 100%;
+    background-color: var(--peacock);
+    height: 4.06rem;
+    top: 0;
+}
+
+.navbar-comp ul {
+    float: right;
+    margin-right: 2rem;
+}
+
+.logo-C {
+    font-size: 34px;
```

Comment:
Try rem ?

Reply:
I have updated unit,thanks for the review